resources/assets/less/bem/notification-popup-item.less

```diff
@@ -82,11 +82,16 @@
     }
   }
 
+  &__link {
+    .full-size();
+  }
+
   &__main {
     border-radius: 0 @border-radius-large @border-radius-large 0;
     flex: 1;
     display: flex;
     min-width: 0;
+    pointer-events: none;
```

Comment:
`__unread-stripe` needs one as well (the one that appears on notifications history page). Or maybe wrap the whole thing (cover, main, stripe) in another div